testing/integration/test/bin/load-zork.sh

```diff
@@ -48,8 +48,8 @@ while getopts b:r:nlpvz:h? opt; do
     esac
 done
 
-
 # Overlap X startup with our download and build.
+rm -f rm /tmp/.X10-lock
```

Comment:
hmm, is `pkill` available?  It'd be nice to make sure any Xvfbs are actually dead -- in case the script's run twice on the same container.  If not:
```bash
ps -a | /bin/grep Xvfb  | awk -e '{print $1}' | xargs kill -9
```